### DISCUSSION TITLE
登录管理界面导入json文件长时间无反应

### DISCUSSION THREAD

**Discussion by QUINCYG6** [mikumifa/biliTickerBuy#291] [提问专区]
![长时间无法登录](https://github.com/mikumifa/biliTickerBuy/assets/174239579/f3c2f8bb-1a21-4aea-8772-d61be09b4bdf)

**Comment by mikumifa** [comment]
重启，试一试

**Reply by mikumifa** [reply]
exe重启，不是电脑

**Reply by QUINCYG6** [reply]
重新导入json到_internal目录下重启exe后已解决，谢谢！
